### DISCUSSION TITLE
How can remove deployment.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#85623] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I  have repositories, where there lots of deployment history. I want to delete it.

**Comment by nickkipshidze** [comment]
It appears you can use GitHub CLI for that too:
```shell
gh api \
  --method DELETE \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /repos/OWNER/REPO/deployments/DEPLOYMENT_ID
```

Or curl:
```shell
curl -L \
  -X DELETE \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/deployments/DEPLOYMENT_ID
```

This may help: [https://docs.github.com/en/rest/deployments/deployments](https://docs.github.com/en/rest/deployments/deployments?apiVersion=2022-11-28#delete-a-deployment)

**Reply by azupworkuser** [reply, marked as answer]
curl -L \
  -X DELETE \
  -H "accept: application/vnd.github+json" \
  -H "authorization: Bearer <token>" \
  -H "x-github-api-version: 2022-11-28" \
  https://api.github.com/repos/owner/repo/deployments/ID

**Comment by redmimehedi0-ux** [comment]
sir, i don,t know this developers.
plaice remove. 
hakar this gay porconal harjment.
